terraform/aws/database-factory-postgresql/variables.tf

```diff
@@ -8,6 +8,11 @@ variable "db_id" {
   type    = string
 }
 
+variable "region" {
```

Comment:
@stafot We don't need to hardcode a region for that because we already export the region in the micro service deployment. The service should be independent of region.